.circleci/config.yml

```diff
@@ -23,6 +24,47 @@ workflows:
              ignore: /.*/
 
 jobs:
+  contract_atomic_swap:
+    docker:
+      - image: rust:1.44.1
+    working_directory: ~/project/contracts/atomic-swap
```

Comment:
please rename dir to cw20-atomic-swap (both here and the actual directory)

It will be the same contract that supports both native and cw20 (just like cw20-escrow), and nice to keep the link there